geolocation-API/iframe.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
```

Comment:
I've seen most test suites create a `resources` subdirectory for non-test files like this.

Reply:
Good suggestion. Will move.